docs/source/app/phasechange.rst

```diff
@@ -58,13 +60,19 @@ Amorphous frame:
 
 .. figure:: ./images/phasechange/pickam.PNG
 
-
+.. _clustering:
 
 Clustering
 ~~~~~~~~~~
-To further categorize the crystalline regions according to their lattice orientation, clustering, based on non-zero diffraction peaks positions, can be used.
-The scripting interface allows to cluster membrane, amorphous and crystalline regions with different lattice orientation in a more efficient way, taking into account regions of interests.
-To look at full analysis `follow this link to a Jupyter notebook <pcmclustering.ipynb>`_.
+
+*This feature was added in version 0.3.0.dev0*
```

Comment:
I missed this when reviewing the PR, but: there are sphinx built-in directives for this, like [`:: versionadded`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded). We should use them in the future :grin: